loader/loader.go

```diff
@@ -350,6 +358,7 @@ func NewLoader(cfg *config.SubTaskConfig) *Loader {
 		tableInfos: make(map[string]*tableInfo),
 		workerWg:   new(sync.WaitGroup),
 		pool:       make([]*Worker, 0, cfg.PoolSize),
+		tctx:       tcontext.Background().WithLogger(log.With(zap.String("task", cfg.Name), zap.String("unit", "load"))),
```

Comment:
Here we create the loader's `tctx` from `context.Background`, but some function such as `func (l *Loader) Process(ctx context.Context, pr chan pb.ProcessResult)` uses the `context` from upper call. So it is confused when we can use Context stored in `loader.tctx`.
For example, if we cancel a context in subtask level, then the `loader.tctx` can't get the notification because its parent `context.Background` is not canceled.

Reply:
But our code implementation is it, like the way we use `context`

Reply:
refine the way to use context later